### PR TITLE
Feature Product page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Added Product page with
- Navbar
- Sidebar with filter component
- Cards

### CODE REVIEW

File: css/product-page.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+.products-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.card {
+    background-color: #EEE;
+    width: 14rem;
```
Comment (sushil-mod) [inline css/product-page.css:66]: use color variable

File: css/product-page.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  .card-price .og-price {
+    text-decoration: line-through;
+    color: grey;
+  }
+  .card-price .disc-percent {
+    color: var(--color-success);
+  }
+  .card button {
+    border: none;
+    background-color: black;
```
Comment (sushil-mod) [inline css/product-page.css:128]: same one

File: css/product-page.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  .card-image img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+  }
+  .card-badge {
+    margin-top: 1rem;
+    background-color: var(--color-primary);
+    color: black;
+    position: absolute;
```
Comment (sushil-mod) [inline css/product-page.css:86]: here  also same one

**Review by sushil-mod — APPROVED** [review]
added some comment go through it , then good to merge

File: css/product-page.css (hunk 3 above)
Comment (abhijitdotsharma) [reply css/product-page.css:86]: I keep forgetting about the color, thanks for pointing out.

File: css/product-page.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    width: 14rem;
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+  }
+  
+  .card-image {
+    position: relative;
+    width: 100%;
+    height: 280px;
```
Comment (neetukumari4858) [inline css/product-page.css:75]: Try to use rem in the Place of px.

**Review by neetukumari4858 — APPROVED** [review]

**Pull request merged**